op-deployer/pkg/deployer/integration_test/apply_test.go

```diff
@@ -611,3 +612,87 @@ func TestApplyGenesisStrategy(t *testing.T) {
 		})
 	}
 }
+
+func TestInvalidL2Genesis(t *testing.T) {
+	op_e2e.InitParallel(t)
+
+	lgr := testlog.Logger(t, slog.LevelDebug)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	depKey := new(deployerKey)
+	l1ChainID := big.NewInt(77799777)
+	dk, err := devkeys.NewMnemonicDevKeys(devkeys.TestMnemonic)
+	require.NoError(t, err)
+
+	l2ChainID1 := uint256.NewInt(1)
+
+	deployerAddr, err := dk.Address(depKey)
+	require.NoError(t, err)
+
+	loc := localArtifactsLocator(t)
+
+	// these tests were generated by grepping all usages of the deploy
+	// config in L2Genesis.s.sol.
+	tests := []struct {
+		name      string
+		overrides map[string]any
+	}{
+		{
+			name: "proxy admin owner not set",
```

Comment:
Can we clarify whether this is the L1 or L2 proxy admin owner? Assuming it’s L1, should there be another test for the L2 proxy admin owner?

Reply:
This is all L2. Will clarify.